modules/nw-scaling-ingress-controller.adoc

```diff
@@ -0,0 +1,93 @@
+// Module filename: nw-scaling-ingress-controller.adoc
+// Module included in the following assemblies:
+// * networking/ingress-controller-configuration.adoc
+
+[id="nw-ingress-controller-configuration-{context}"]
+= Scaling an ingress controller
+
+An ingress controller can be manually scaled up or down based on
+requirements such as the need to increase throughput. `oc`
+commands are used to scale the `IngressController` resource.
+
+.Procedure
+. View the current number of replicas of the ingress controller resource.
++
+----
+$ name="foo"  <1>
```

Comment:
Would it be simpler to make the example in terms of the `default` ingresscontroller to avoid the local and references to the local?

Reply:
Done